### PR TITLE
Request all the things

### LINKED ISSUE #287: Display request buttons in components lists
**Acceptance criteria**

In displays where lists of components appear:

* [x] Display a "Request" button for each component with containers
* [x] Clicking a request button should pop up a request form with the right info 

**Questions**

* Should this appear in collection's inventory view as well as the tabular component display (#231)?
* Is this expected to be achieved how do we best select multiple components for requests? Check boxes or something similar?

**Mockup**

arclight-mockups-mvp

### PR DESCRIPTION
**Pull request by mejackreed**
Closes #287 

I recognize that likely there will need to be some design changes follow on for accommodation of this button.

request-everything

### CODE REVIEW

File: solr/conf/solrconfig.xml [hunk 1]
```diff
@@ -122,7 +122,8 @@
          ref_ssm,
          component_level_isim,
          parent_access_restrict_ssm,
-         parent_access_terms_ssm
+         parent_access_terms_ssm,
+         containers_ssim
```
Comment (drh-stanford) [inline solr/conf/solrconfig.xml]: `containers_ssim` is already in here on line 106.

Comment (mejackreed) [inline solr/conf/solrconfig.xml]: Nice catch.. pushed an update.

File: app/controllers/concerns/arclight/field_config_helpers.rb [hunk 2]
```diff
@@ -31,6 +32,12 @@ def repository_config_present(_, document)
       document.repository_config.present?
     end
 
+    def item_requestable?(_, options)
+      document = options[:document]
+      # byebug
```
Comment (drh-stanford) [inline app/controllers/concerns/arclight/field_config_helpers.rb]: can we remove this debugging statement?

Comment (mejackreed) [inline app/controllers/concerns/arclight/field_config_helpers.rb]: 👍 thanks good catch.. pushed.

**Pull request merged**